contrib/miniupnpd/update.py

```diff
@@ -0,0 +1 @@
+pattern = r"/refs/tags/miniupnpd_(\d+_\d+_\d+)\.tar\.gz"
```

Comment:
also override fetch_versions to map the _ back to . (see e.g. opensubdiv update.py)